resources/assets/js/compress-image.js

```diff
@@ -110,8 +110,8 @@ const CompressImage = (
             maxHeight: $maxHeight,
             minWidth: $minWidth,
             minHeight: $minHeight,
-            width: $width ?? undefined,
-            height: $height ?? undefined,
+            width: isNaN(parseInt($width)) ? undefined : $width,
```

Comment:
It's actally an empty string which gets passed in from [here](https://github.com/ArkEcosystem/laravel-foundation/blob/34678a7789e40c73e92c2ee945ed9b1eaa1ccf84/resources/views/inputs/upload-image-collection.blade.php#L29-L30) which is why ?? does not pick it up as not being set.

Reply:
that confuses me a bit 🤔 will check if there was a reason to force the strings there or if it could work without so we get a proper null/undefined value to work with

Reply:
Will also take a look but IIRC upload-image-collection complains of you try to pass ‘null’ into the JS component which I guessed was why it was being converted into a string.